libraries/Microsoft.Bot.Builder.Dialogs.Adaptive.Testing/Mocks/MockSkillBotFrameworkClient.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright(c) Microsoft Corporation.All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder.Skills;
+using Microsoft.Bot.Schema;
+
+namespace Microsoft.Bot.Builder.Dialogs.Adaptive.Testing.Mocks
+{
+    /// <summary>
+    /// BotFrameworkClient mock.
+    /// </summary>
+    public class MockSkillBotFrameworkClient : BotFrameworkClient
```

Comment:
I'm not sure this should be in Microsoft.Bot.Builder.Dialogs.Adaptive.Testing?  This class seems more specific to skill testing.  Can it be moved into Microsoft.Bot.Builder.Dialogs.Adaptive.Tests?